packages/graph/hash_graph/lib/graph/src/lib.rs

```diff
@@ -55,7 +55,6 @@
               usage"
 )]
 #![expect(
-    clippy::cast_sign_loss,
```

Comment:
How did this disappear?

Reply:
We don't return the entity metadata from the database when moving an entity to the historic table anymore, so the version of the VersionedUri does not need to be retrieved.

Reply:
Can you point me to the specific line? I tried looking for it but didn't find it

Reply:
In [packages/graph/hash_graph/lib/graph/src/store/postgres/mod.rs](https://github.com/hashintel/hash/pull/1544/files#diff-8f613375ad243c59cb8d23f81c2d7602cc3264296def6961c83d976a689e0f84), I removed line 841
```diff
- let entity_type_id = VersionedUri::new(base_uri, historic_entity.get::<_, i64>(4) as u32);
```

This line is only called in `update_entity()`. This function is going to be changed soon btw.

Reply:
Cheers